### ISSUE TITLE
Adding a Lite flag to tf.keras.applications.EfficientNetBX

### ISSUE THREAD

**Issue by hhass** [tensorflow/tensorflow#45091]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 2.4.0-rc2
- Are you willing to contribute it (Yes/No): No

**Describe the feature and the current behavior/state.**
Currently, EfficientNetB0 to B7 are available via keras.applications.
In May, tensorflow released a [blog entry](https://blog.tensorflow.org/2020/03/higher-accuracy-on-vision-models-with-efficientnet-lite.html) that 'Lite' variants of EfficientNet are available on [tensorflow/tpu](https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet/lite). 
The models can in addition be used from tensorflow [Hub](https://tfhub.dev/tensorflow/efficientnet/lite0/classification/2) and tfLite model maker. Yet, these models are created as TF1 graphs and cannot be readily used for fine-tuning etc. in tensorflow 2.X.

Is it possible to introduce a flag in tf.keras.applications.EfficientNet that will load the 'lite' variant, similar to the 'minimalistic' flag of MobileNetV3?

**Will this change the current api? How?**
Yes, tensorflow/python/keras/applications/efficientnet.py would get an additional flag that will load the corresponding weight file, and will remove squeeze&excite modules and substitute swish by relu6 

**Who will benefit with this feature?**
Everyone who wants to use EfficientNet Lite in tensorflow2, for fine-tuning or layer-wise custom actions.

**Any Other info.**

**Comment by hfahrudin** [comment]
Is this feature doable? if so, i would like to tried to make it.

**Comment by jdduke** [comment]
Over to @lintian06 for further comment.

**Comment by sebastian-sz** [comment]
@lintian06 
I managed to modify the existing code to add option for EfficientNet Lite variant architecture and to convert the weights via utilility script provided in [Tensorflow repo](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/applications/efficientnet_weight_update_util.py).

Are you open for PRs regarding this issue?

**Comment by mohantym** [comment]
Hi @hhass !

I am doubtful that adding a lite variant to tf.keras.application will be worth the effort as both lite model and keras model has different model architecture( one is  very optimized and compressed to run on edge devices and later is not). 

Thank you!

**Comment by pjpratik** [comment]
Closing this as stale. Please reopen if this is still a valid request.
